script/push_docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Copyright 2021 Adevinta
+
+# This script tags and pushes a locally existent docker image to a specified new
+# tag. It receives two parameters, the first one specifies the tag of the local
+# image and the second one the tag to push the image to. It expects the env
+# variables: DOCKER_USERNAME and DOCKER_PASSWORD to be set. Those credentials
+# must belong to a valid user with permission to push to the docker image
+# registry the script will push the image to.
+
+set -eu
+
+if [ $# -ne 2 ]; then
+    echo "usage: $0 <local_image> <push_image>" >&2
```

Comment:
Given the name of the variables (`LOCAL_TAG` and `PUSH_TAG`) I'd probably change this usage message to:

```sh
    echo "usage: $0 <local_tag> <push_tag>" >&2
```

Reply:
okey makes sense.